dts/arm/st/g0/stm32g070X6.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I haven't seen any reference to this one (only STM32G070CB STM32G070KB STM32070RB)
Do you have any link ?

Reply:
I have accidentally duplicated that from 71x6.